common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -622,16 +619,13 @@ def get_problem_html(self, encapsulate=True):
 
         html = self.remove_tags_from_html(html)
 
-        # The convention is to pass the name of the check button if we want
-        # to show a check button, and False otherwise This works because
+        # The convention is to pass the name of the submit button if we want
+        # to show a submit button, and False otherwise This works because
```

Comment:
This comment doesn't make sense anymore, correct? We always show the Submit button, just sometimes disable it.